miio/gateway.py

```diff
@@ -646,71 +646,112 @@ class GatewayLight(GatewayDevice):
     """Light controls for the gateway."""
 
     @command()
-    def get_night_light_rgb(self):
-        """Unknown."""
-        # Returns 0 when light is off?"""
-        # looks like this is the same as get_rgb
-        # id': 65064, 'method': 'set_night_light_rgb', 'params': [419407616]}
-        # {'method': 'props', 'params':
-        # {'light': 'on', 'from.light': '4,,,'}, 'id': 88457} ?!
-        return self._gateway.send("get_night_light_rgb")
+    def rgb_status(self):
+        """Get current status of the light (always correct)."""
+        # Returns {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)} when light is off
+        state_int = self._gateway.send("get_rgb").pop()
+        brightness = int_to_brightness(state_int)
+        rgb = int_to_rgb(state_int)
+        if brightness > 0:
+            is_on = True
+        else:
+            is_on = False
```

Comment:
This can be simplified by using `is_on = brightness > 0`. Same also below for the night light.

Reply:
done, thanks!